app/components/modal/PrivateCloudDelete.tsx

```diff
@@ -182,19 +183,14 @@ export default function PrivateCloudDeleteModal({
             Deleting...
           </button>
         ) : (
-          <button
+          <Button
+            color="danger"
             disabled={isDisabled || deletionCheckData !== 'OK_TO_DELETE'}
             type="button"
```

Comment:
since button is the default type of the component Button, let's remove it.

Reply:
okay